.pylintrc

```diff
@@ -53,7 +53,7 @@ persistent=yes
 
 # Minimum Python version to use for version dependent checks. Will default to
 # the version used to run pylint.
-py-version=3.7
+py-version=3.8
```

Comment:
is `.pylintrc` still required ? (I'm not an expert)

Reply:
No, it is not due to introducing `ruff`.

Reply:
It seems simplest to remove this file then

Reply:
Good to me

Reply:
Resolved via 6a5d10c794099fb5ba2974e171987c5af956baf6